linera-service/src/benchmark.rs

```diff
@@ -120,7 +120,9 @@ async fn benchmark_with_fungible(
     }
 
     info!("Building the fungible application bytecode.");
-    let path = Path::new("examples/fungible").canonicalize().unwrap();
+    let path = Path::new("examples/fungible").canonicalize().context(
+        "are you sure you are running the binary from the `linera-protocol` repository directory?",
```

Comment:
Maybe: "`linera-benchmark` is meant to run from the root of the `linera-protocol` repository"

Now I understand why the feature is helpful btw (for people doing `cargo install linera-service`)